functions/core.sh

```diff
@@ -26,6 +26,13 @@ function unmount() {
 	sudo umount rootfs
 }
 
+function fs_features() {
+	for img in system.img rootfs.img; do
+		sudo resize2fs -s $img
```

Comment:
isn't this run twice for system.img now?

Reply:
It's run once for rootfs.img and once for system.img

Reply:
Is it not the second time after this one: https://github.com/JBBgameich/halium-install/blob/master/functions/core.sh#L22

Reply:
Ah that's what you mean. Yeah, although run with different arguments it would be better to merge both commands. I'm working on it